packages/nextjs/components/impact-vector/ImpactVectorListCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { SelectedVector } from "~~/app/types/data";
+import { useGlobalState } from "~~/services/store/store";
+
+const ImpactVectorListCard = ({
+  title,
+  description,
+  vectors,
+}: {
+  title: string;
+  description: string;
+  vectors: SelectedVector[];
+}) => {
+  const { setSelectedVectors } = useGlobalState();
+
+  return (
+    <div
+      onClick={() => setSelectedVectors(vectors)}
+      className="mr-1  cursor-pointer rounded-xl text-sm border-[0.2px] border-secondary-text/50 p-4 bg-base-300 my-2"
```

Comment:
In the absence of designs I think what you have here is fine for now though it might be nice to add the creator address to the bottom of the card similar to the Impact Vector cards having '@OSO' at the bottom.

Reply:
the vector list only contains the creator address, is that ok?

Reply:
![Screenshot from 2024-04-06 23-51-31](https://github.com/BuidlGuidl/impact-calculator/assets/64108987/9f7ab7b0-19bf-4e6d-b47c-979c0e03b6be)

Reply:
Lets make it say,

`Creator: <full creator address>`
That address might look best if you use the Address component that shows the ENS + profile pictures.

Reply:
![Screenshot from 2024-04-07 14-35-51](https://github.com/BuidlGuidl/impact-calculator/assets/64108987/04d78fab-8b69-4c95-a3e9-02df06ce5cc4)